### ISSUE TITLE
`awesome-toc` claims footnote subtitles are missing from ToC

### ISSUE THREAD

**Issue by sancarn** [sindresorhus/awesome-lint#200]
Our footnotes has the following:

```md
## Footnotes

### Symbology
```

The linter reports:

```
34:3  ToC missing item for "Symbology"                  remark-lint:awesome-toc
```

Feel like this is definitely a bug. Linting should just stop at Footnotes and not continue imo, unless footnotes should be in ToC as a main title, i.e:

```
- ... my awesome sections ...
- [FootNotes](#footnotes)
    - [Symbology](#symbology)
```

or

```
- ... my awesome sections ...
```

Current requirement appears to be:

```
- ... my awesome sections ...
- [Symbology](#symbology)
```
